src/audio_player.rs

```diff
@@ -0,0 +1,90 @@
+use crate::sync_wrapped;
+use bevy::prelude::{AudioSinkPlayback, Component, Vec3};
+
+#[derive(Component)]
+pub struct AudioListener {
+    pub previous_position: Vec3,
+}
```

Comment:
This could be a `Bundle` using `Transform`. That way we can implement [direction and elevation](https://www.fmod.com/docs/2.02/studio/parameters-reference.html#direction) in the future.

Reply:
Direction and elevation are part of the built-in parameters, they will get updated automatically based on the 3dAttributes we set. For something like directivity we either have to use user parameters (aka local parameters) or use 3rd party plugins afaik